test/command_callback/test_racket_raco_command_callback.vader

```diff
@@ -0,0 +1,10 @@
+Before:
+  call ale#assert#SetUpLinterTest('racket', 'raco')
+
+After:
+  call ale#assert#TearDownLinterTest()
+
+Execute(The default command and executable should be correct):
+  AssertLinter 'raco', ale#Escape('raco') . ' expand %s'
```

Comment:
```suggestion
  AssertLinter 'raco', 'raco expand %s'
```

You can remove `ale#Escape` on the right here. The executable for the linter isn't escaped because it isn't configurable, it's just written into the command string.

Reply:
Remove the extra blank lines at the end of the file.